packages/gatsby-plugin-manifest/src/__tests__/gatsby-node.js

```diff
@@ -102,8 +102,11 @@ describe(`Test plugin manifest options`, () => {
       theme_color: `#a2466c`,
       display: `standalone`,
     })
-    const [filePath, contents] = fs.writeFileSync.mock.calls[0]
-    expect(filePath).toEqual(path.join(`public`, `manifest.webmanifest`))
+    const contents = fs.writeFileSync.mock.calls[0][1]
+    expect(fs.writeFileSync).toHaveBeenCalledWith(
+      path.join(`public`, `manifest.webmanifest`),
+      expect.anything()
+    )
     expect(sharp).toHaveBeenCalledTimes(0)
     expect(contents).toMatchSnapshot()
```

Comment:
I'm not sure how to do snapshot matching in arguments, that would remove this .mock call